### ISSUE TITLE
Please upload BOM file

### ISSUE THREAD

**Issue by DRCRecoveryData** [dotPY-hax/jhackulator#2]
Hi, could you export bom file and compoments soldering position on pcb?

**Comment by dotPY-hax** [comment]
The BOM and component placement is in the gerber.zip

**Comment by DRCRecoveryData** [comment]
> The BOM and component placement is in the gerber.zip

Can i ask you does the ch340n working same as original FTDI function?

**Comment by DRCRecoveryData** [comment]
> The BOM and component placement is in the gerber.zip

Hi,

Does uart passthough working on your pcb?

**Comment by dotPY-hax** [comment]
Yes it does. Make sure to use the correct baud rate and voltage.
